### DISCUSSION TITLE
Change date separator character

### DISCUSSION THREAD

**Discussion by ilbrando** [js-fns/date-fns#2209] [Q&A]
In Denmark (locale: da) it is common to use both "/" and "-" as separator in dates. The current implementation uses "/", so dates are formatted like 24/12/2020. Is there a way to change this so dates are formatted 24-12-2020?

**Comment by anilloutombam** [comment]
You can easily modify by providing like this 
````
const PARSED_DATE_FORMAT1 = 'MM/dd/yy';
const PARSED_DATE_FORMAT2 = 'MM/dd/yyy';
const PARSED_DATE_FORMAT3 = 'MM-dd-yy';
````
and pass it to your props
